api/nameguard/nameguard.py

```diff
@@ -163,6 +170,9 @@ def inspect_name_sync(
 
         logger.debug(f"[inspect_name] name: '{name}'")
 
+        if len(name) > INSPECTABLE_NAMES_LENGTH:
```

Comment:
@Carbon225 What about collection generation? Can the result be null?

Reply:
@djstrong Hey, I missed this comment from before. Is there any remaining open question here?

Reply:
No, now we don't return null but uninspected report.